packages/stacked_core/README.md

```diff
@@ -0,0 +1,39 @@
+<!-- 
```

Comment:
Please update to 


```md
# Stacked Core

This package contains all the code shared between our flutter and pure dart packages to avoid including flutter where it's not required.
```

Reply:
Done